app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -106,5 +107,8 @@ class MapViewCustomizationActivity : AppCompatActivity() {
     super.onDestroy()
     mapView.onDestroy()
     customMapView.onDestroy()
+    // Restore default resource Settings
```

Comment:
I'd drop a broader comment notifying that if not restore defaults explicitly - all further created `MapView`s will use such setup silently.

Reply:
updated doc